backend/deploy

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+if [[ $PWD != */backend ]]; then
```

Comment:
Then you need to test that the pushd works.
I could extract the directory from $0 but that's sorta flaky, too. See http://mywiki.wooledge.org/BashFAQ/028
Requiring they be in the backend doesn't seem like a high hurdle.